### PR TITLE
Release notes for 3.4.11 release

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -60,6 +60,7 @@ pages:
 - Release Information:
   - 'Release Note Overview': 'release/notes.md'
   - 3.4 Release Notes:
+    - 'OSG Release 3.4.11': 'release/3.4/release-3.4.11.md'
```
Comment (brianhlin) [inline mkdocs.yml]: This is causing the broken build, should be `relaese-3-4-11.md`

**Comment by brianhlin** [comment]
LGTM but I would like @DrDaveD to review the language for Singularity

File: docs/release/3.4/release-3-4-11.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+**Release Date**: 2018-05-01
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   [Singularity 2.5.0](https://github.com/singularityware/singularity/releases/tag/2.5.0)
+    - This release includes fixes for several high and medium severity security issues.
+      It also contains a whole slew of bug fixes including the much awaited docker aufs whiteout file fix.
```
Comment (DrDaveD) [inline docs/release/3.4/release-3-4-11.md]: Actually, "whole slew of" is probably too informal, and maybe also overstates it.  How about just "a number of".

Comment (timtheisen) [reply docs/release/3.4/release-3-4-11.md]: I just cut and pasted that out of their announcement.

Comment (DrDaveD) [reply docs/release/3.4/release-3-4-11.md]: Alright if you must, but we don't have to be as informal as they are.

Comment (timtheisen) [reply docs/release/3.4/release-3-4-11.md]: Not a problem, I update it to make us look more classy.

**Pull request merged**